### PR TITLE
Fix add user dialog

### LINKED ISSUE #33951: [Bug]: NC 25 New User from missing better visible background shadow or outline
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

image

### Steps to reproduce

1. Users
2. Create new user
3.

### Expected behavior

Should have a outline around it 

### Installation method

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database engine version

_No response_

### Is this bug present after an update or on a fresh install?

_No response_

### Are you using the Nextcloud Server Encryption module?

_No response_

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
Na
```

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by CarlSchwan**
- Add label instead of placeholder for accessibility
- Port to new vue components

### CODE REVIEW

**Comment by szaimen** [comment]
the multiselect looks strange but I guess it will be improved with the next beta release of vue 7.0, correct?

**Comment by Pytal** [comment]
Let's get this in after https://github.com/nextcloud/server/pull/34044

File: apps/settings/src/components/UserList.vue [hunk 1]
```diff
@@ -41,39 +42,43 @@
 					class="modal__item"
 					name="username"
 					pattern="[a-zA-Z0-9 _\.@\-']+"
-					required
-					type="text">
-				<input id="newdisplayname"
-					v-model="newUser.displayName"
-					:placeholder="t('settings', 'Display name')"
+					required>
+				</NcTextField>
```
Comment (Pytal) [inline apps/settings/src/components/UserList.vue:46]:
```suggestion
					required />
```
**suggestion (non-blocking)**

File: apps/settings/src/components/UserList.vue [hunk 2]
```diff
@@ -41,39 +42,43 @@
...
+					:labelVisible="true"
 					autocapitalize="none"
 					autocomplete="off"
 					autocorrect="off"
 					class="modal__item"
-					name="displayname"
-					type="text">
-				<input id="newuserpassword"
+					name="displayname">
+				</NcTextField>
```
Comment (Pytal) [inline apps/settings/src/components/UserList.vue:56]:
```suggestion
					name="displayname" />
```
**suggestion (non-blocking)**

File: apps/settings/src/components/UserList.vue [hunk 3]
```diff
@@ -41,39 +42,43 @@
...
 					autocomplete="new-password"
 					autocorrect="off"
 					class="modal__item"
-					name="password"
-					type="password">
-				<input id="newemail"
-					v-model="newUser.mailAddress"
-					:placeholder="t('settings', 'Email')"
+					name="password">
+				</NcPasswordField>
```
Comment (Pytal) [inline apps/settings/src/components/UserList.vue:69]:
```suggestion
					name="password" />
```
**suggestion (non-blocking)**

File: apps/settings/src/components/UserList.vue [hunk 4]
```diff
@@ -41,39 +42,43 @@
...
+					:label="t('settings', 'Email')"
+					:labelVisible="true"
 					:required="newUser.password==='' || settings.newUserRequireEmail"
 					autocapitalize="none"
 					autocomplete="off"
 					autocorrect="off"
 					class="modal__item"
 					name="email"
 					type="email">
+				</NcTextField>
```
Comment (Pytal) [inline apps/settings/src/components/UserList.vue:81]:
```suggestion
					type="email" />
```
**suggestion (non-blocking)**

**Review by Pytal — APPROVED** [review]

**Comment by PVince81** [comment]
- [ ] BUG: dialog doesn't close after creating a user, it just empties itself (same with an without the admin password prompt)

**Comment by szaimen** [comment]
> * [ ]  BUG: dialog doesn't close after creating a user, it just empties itself (same with an without the admin password prompt)

Maybe something wrong with resetting the focus or something?

---

Also, can we remove the custom multiselect styling and check if it improves the look?

**Comment by szaimen** [comment]
@CarlSchwan what is missing here?

**Comment by PVince81** [comment]
@Pytal please take over when the time is right as it touches accessibility

**Comment by szaimen** [comment]
Now that we updated nextcloud/vue miltiple times in the meantime, maybe even a rebase already makes it work...

**Comment by szaimen** [comment]
I rebase this and pushed the changes to https://github.com/nextcloud/server/pull/35898

**Pull request closed without merging**